extensions/cornerstone/src/init.tsx

```diff
@@ -230,41 +237,41 @@ export default async function init({
    * active tools in the toolbar, and call any commands registered in the
    * toolbar service with a callback to re-enable on displaying the viewport.
    */
-  const toolbarEventListener = evt => {
-    const { element } = evt.detail;
-    const activeTools = toolbarService.getActiveTools();
+  // const toolbarEventListener = evt => {
```

Comment:
Are you planning on leaving this commented out for now?  Would like to see it removed in the final commit.